# Accelerometer [m/s^2]

geometry_msgs/Vector3 accel

# Gyroscope [rad/s]

geometry_msgs/Vector3 gyro

# Internal Temperature [Â°C]
float32 temp

# delta velocity output [m/s2]

geometry_msgs/Vector3 delta_vel

# delta angle output [rad/s]

geometry_msgs/Vector3 delta_angle
